.gitignore

```diff
@@ -1,2 +1,3 @@
 **/target
-.idea
\ No newline at end of file
+.idea
+/migration/liquibase.properties
```

Comment:
this property might be public, but use simple credentials (admin:admin) in your local tools